test/fixtures/projects.yml

```diff
@@ -145,3 +145,9 @@ calculation_test_1:
 calculation_test_2:
   title: Project for invalid formula calculation tests
   user: boxes
+
+weather_data:
+  title: Weather Data
+  user: patson
+  content: Simple project to test Formula Fields from uploaded data.
+  
```

Comment:
I think it's not counting it as a new line because there's whitespace in it. I need to remember to set VS Code to always append new lines on save tomorrow.